torch/csrc/jit/runtime/static/generated_ops.cpp

```diff
@@ -3465,7 +3464,7 @@ REGISTER_OPERATOR_FUNCTOR(
           const auto ignore_index = p_node->Input(5).toInt();
           const auto& total_weight = p_node->Input(6).toTensor();
           if (p_node->Output(0).isNone()) {
-            p_node->Output(0) = at::cpu::nll_loss_backward(
+            p_node->Output(0) = at::nll_loss_backward(
```

Comment:
this change is completely unnecessary, I think the old change should have compiled